README.md

```diff
@@ -4,6 +4,16 @@
 
 ## How does it work?
 
+Please refer to the [official documentation](https://docs.newrelic.com/docs/integrations/kubernetes-integration/metadata-injection/kubernetes-apm-metadata-injection) to learn the reasoning behind this project.
+
+## Setup
+
+Please refer to the [setup instructions in the official documentation](https://docs.newrelic.com/docs/integrations/kubernetes-integration/metadata-injection/kubernetes-apm-metadata-injection#install).
+
+## Development
+
+### How does it work?
+
 New Relic requires the following environment variables to identify Kubernetes objects in the APM agents:
```

Comment:
I would change this sentence to: New Relic APM agents requires the following environment variables to provide Kubernetes object information in the context of... (it's a mix with the description in the official docs site).